code/modules/mob/language.dm

```diff
@@ -426,6 +426,50 @@
 	key = "0"
 	syllables = list ("honk","squeak","bonk","toot","narf","zub","wee","wub","norf")
 
+//Out of system languages (non-standard, but still common)
+/datum/language/com_srus
+	name = "Neo-Russkiya"
+	desc = "Neo-Russkiya, a bastard mix of Gutter, Sol Common, and old Russian. Maintained primarily by USSP citizens and fringe traders."
+	speech_verb = "articulates"
+	whisper_verb = "mutters"
+	exclaim_verb = "exaggerates"
+	colour = "com_srus"
+	key = "<"
+	space_chance = 65
+	english_names = 1
+	syllables = list("dyen","bar","bota","vyek","tvo","slov","slav","syen","doup","vah","laz","gloz","yet",
+					 "nyet","da","sky","glav","glaz","netz","doomat","zat","moch","boz",
+					 "comy","vrad","vrade","tay","bli","ay","nov","livn","tolv","glaz","gliz",
+					 "ouy","zet","yevt","dat","botat","nev","novy","vzy","nov","sho","obsh","dasky",
+					 "key","skey","ovsky","skaya","bib","kiev","studen","var","bul","vyan",
+					 "tzion","vaya","myak","gino","volo","olam","miti","nino","menov","perov",
+					 "odasky","trov","niki","ivano","dostov","sokol","oupa","pervom","schel",
+					 "tizan","chka","tagan","dobry","okt","boda","veta","idi","cyk","blyt","hui","na",
+					 "udi","litchki","casa","linka","toly","anatov","vich","vech","vuch","toi","ka","vod")
+
+//Trad space russkiya for event usage, like USSP officials or operatives. Child of com_srus since same syllables.
+/datum/language/com_srus/trad_srus
+	name = "Traditional Russkiya"
+	desc = "A more formalized form of Russkiya that attempts to (poorly) adhere to old Russian grammar and rules. The official langauge of the USSP."
+	colour = "trad_srus"
+	key = ">"
+	flags = RESTRICTED
+
+/datum/language/elysian
+	name = "Elysian"
+	desc = "A secretive winding language rarely spoken outside the Elysium Republic. It's often considered rude to speak in this tongue due to the Elysium Republic's stance on non-humans."
+	speech_verb = "dulciquents"
```

Comment:
My googling seems to imply that this (along with `interpollates` and `sussurates`) is spelt wrongly. Are you sure these are correct?

Reply:
I can't believe I missed that, my bad! Thanks for pointing that out.